### PR TITLE
feat(cdsctl): action usage command

### PR DESCRIPTION
**Pull request by codebfu**
Signed-off-by: Gaultier HUBERT <gaultier.hubert@corp.ovh.com>

1. Description
1. Related issues
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#12587.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✘  
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✔

File: sdk/action.go [hunk 1]
```diff
@@ -46,6 +46,7 @@ type Action struct {
 	Description string `json:"description" yaml:"desc,omitempty" db:"description"`
 	Enabled     bool   `json:"enabled" yaml:"-" db:"enabled"`
 	Deprecated  bool   `json:"deprecated" yaml:"-" db:"deprecated"`
+	ParentName  string `json:"group_name,omitempty" yaml:"-" db:"group_name"`
```
Comment (richardlt) [inline sdk/action.go]: To remove ?

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#12641.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✘  
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#12641.1 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✔ 
* Package
  * Docker ✔ 
  * Docs ✔ 
* Integration
  * Docker Compose Tests ✔ 
  * Helm Tests -   
  * Standalone Tests ✘

**Pull request merged**